crates/biome_cli/src/diagnostics.rs

```diff
@@ -231,9 +232,14 @@ pub struct IncompatibleEndConfiguration {
 #[diagnostic(
     category = "internalError/io",
```

Comment:
I think this is not an internal error, what is the appropriate category?

Reply:
We could use the category of the command being run. E.g. `biome check` -> `check`